_posts/2019-01-22-providing-feedback-to-bots.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Why and How do we provide feedback to the bots?"
+date: 2019-01-22 7:30:00 +0000
+comments: false
+---
+
+### Why do we need to feedback to bots? 
+
+We send feedback to the bots for us to know which all reports have already been handled. We also collect some statistics and  improve the filters based on this feedback. Few of the bots are built on ML and hence they need feedback to improve. 
+
+The continual improvement of the bots is one way to see how your feedback has affected the bot. Soon we are planning to utilize SOPlotics to plot nice graphs and figures for the same. 
+
+The bots do log these data. Some bots, like Guttenberg, send the data to a dashboard, like CopyPastor, which stores the data on its behalf. 
+
+
+### What all bots  take feedback? 
+
+ - Belisarius posts possible vandalized posts and takes `tp` or `fp` as feedback.
+ - Guttenberg posts possible plagiarized answers and takes `k` or `f` (and `tp` or `fp`) as feedback. See [How to react to Guttenberg's reports?](https://github.com/SOBotics/Guttenberg/blob/master/feedback.md)
+ - HeatDetector posts possible inflammatory comments and takes `tp`, `fp`, `fn`, `nc` and `sk` as feedback. See [What all types of feedback does Heat Detector take?](https://github.com/SOBotics/HeatDetector/wiki/Feedback)
+ - Natty posts possible non-answers and takes `tp`, `fp` and `ne` as feedback. See [How do I interact with Natty?](https://natty.sobotics.org/interacting). 
+ - SOCVFinder posts possible duplicate questions and takes `k` or `f` as feedback. 
+
+Additionally, our guest, SmokeDetector, posts possible spam or rude or abusive posts, and it takes `tp` (`k`, `v`), `fp` (`f`) or `naa` (`n`) as feedback via replying to its report messages.
```

Comment:
Add: "See [Feedback Guidance](https://charcoal-se.org/smokey/Feedback-Guidance) on the Charcoal website for details."